Exception/Exception.php

```diff
@@ -47,14 +47,3 @@
  */
 
 class Exception extends \Hoa\Core\Exception { }
-
-}
-
-namespace {
-
-/**
- * Flex entity.
- */
-Hoa\Core\Consistency::flexEntity('Hoa\File\Exception\Exception');
```

Comment:
I can keep it, what I was trying to understand was what the `flexEntity` was doing. Seems like some sort of short cuts. Your suggestion doesn't helped though :-/